### PR TITLE
[bitnami/spark]  Fixed the mount path of shared-certs volume

### PR DESCRIPTION
**Pull request by anebi**
### Description of the change
Changed the mount path of shared-certs volume in Apache Spark chart.

### Benefits

Successful deployment when SSL is enabled for Apache Spark. This change allows Spark to find the certificates in default location where it search them.

### Applicable issues

https://github.com/bitnami/charts/issues/13290 issue.

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by javsalgar** [comment]
Hi!

Thank you so much for the PR! Could you fix the DCO issue?

**Comment by anebi** [comment]
Hi, DCO fixed :)

File: bitnami/spark/templates/statefulset-worker.yaml [hunk 1]
```diff
@@ -182,11 +182,11 @@ spec:
           volumeMounts:
             {{- if .Values.worker.existingConfigmap }}
             - name: config
-              mountPath: '/bitnami/spark/conf/'
+              mountPath: '/opt/bitnami/spark/conf/'
```
Comment (migruiz4) [inline bitnami/spark/templates/statefulset-worker.yaml:185]: Could you please keep this one as `/bitnami/spark/conf/`? This path corresponds to `SPARK_MOUNTED_CONFDIR`, and files from `/bitnami/spark/conf/` will be copied to `/opt/bitnami/spark/conf/`  at runtime.

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Thank you very much for your contribution, could you please take a look at my comment?

File: bitnami/spark/templates/statefulset-worker.yaml (hunk 1 above)
Comment (anebi) [reply bitnami/spark/templates/statefulset-worker.yaml:185]: Yup, let me change that.

File: bitnami/spark/templates/statefulset-worker.yaml [hunk 2]
```diff
@@ -142,7 +142,7 @@ spec:
           - name: certs
             mountPath: /certs
           - name: shared-certs
-            mountPath: /bitnami/spark/conf/certs
+            mountPath: /opt/bitnami/spark/conf/certs
```
Comment (migruiz4) [inline bitnami/spark/templates/statefulset-worker.yaml:145]: Sorry for the inconvenience, but I may have to ask you to also apply my previous suggestion here

**Review by migruiz4 — CHANGES_REQUESTED** [review]

**Review by migruiz4 — APPROVED** [review]
Thank you very much for applying the suggested changes, looks good to me!

**Pull request merged**